README.md

```
@@ -151,8 +153,82 @@ This can occurs if `bundled/libs` is empty. That is the folder where we put your
-# TODO: The maintainer of this repo has not yet edited this file
-
-**Repo Owner** Make sure you update this. As a repository owner you will need to update this file with specific instructions for your extension.
+## TODO
+
+> WARNING  This extension consists of 9677 files, out of which 3 are JavaScript files. For performance reasons,
+> you should bundle your extension: https://aka.ms/vscode-bundle-extension. You should also exclude unnecessary
+> files by adding them to your .vscodeignore: https://aka.ms/vscode-vscodeignore.
+
+### Intermittent
+
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)